.github/workflows/ci.yml

```diff
@@ -113,7 +113,7 @@ jobs:
 
       - name: a11y test
         if: steps.changes.outputs.deploy == 'true'
-        run: yarn workspace @okta/odyssey-storybook test:storybook --url ${{ env.URL_STORYBOOK }}
+        run: yarn workspace @okta/odyssey-storybook test-storybook --url ${{ env.URL_STORYBOOK }}
```

Comment:
Maybe I'm missing something, but does this named script `test-storybook` exist?

Reply:
Correct. `test-storybook` is exported by Storybook.